src/index.html

```diff
@@ -1,13 +1,463 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>THE MET</title>
+    <link rel="shortcut icon" href="images/THE_MET_favicon.svg" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="top__actions">
+          <a
+            href="/"
+            class="header__logo">
+          </a>
+          <div class="header__nav">
+            <a href="tel:+1 212-535-7710" class="icon icon--phone header__call"> </a>
+            <a href="#menu" class="icon icon--menu"></a>
+          </div>
+        </div>
+          
+        <h1 class="header__title">
+          Welcome
+          <br>
+          to The MET
+        </h1>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top__actions">
+            <a href="#" class="header__logo icon"> </a>
+            <a href="#" class="icon icon--cross"></a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About us</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#now-on-view" class="menu__link">Now on view</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#explore-online" class="menu__link">Explore Online</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a href="+1 212-535-7710" class="menu__call menu__call--phone">The Met Fifth Avenue <br> +1 212-535-7710 </a>
```

Comment:
It doesn't work with this href value, try href='tel: +1 212-535..'. Change these for all links with numbers